app/src/main/java/org/oppia/android/app/walkthrough/WalkThroughVerticalScrollListener.kt

```diff
@@ -0,0 +1,10 @@
+package org.oppia.android.app.walkthrough
+
+/** Listener for when an activity should hide and show ProgressBar And HeaderTextView. */
```

Comment:
/** Interface to show or hide the Progress Bar and Header Text. */

Reply:
Can we use abetter naming then HeaderTextView, which resemble more on what's the role of header rather then what it is.

Reply:
@anandwana001 Can you please suggest one?